pkg/base/config.go

```diff
@@ -524,15 +524,15 @@ type RaftConfig struct {
 	// an election. The actual election timeout is randomized by each replica to
 	// between 1-2 election timeouts. This value is inherited by individual stores
 	// unless overridden.
-	RaftElectionTimeoutTicks int
+	RaftElectionTimeoutTicks uint64
 
 	// RaftReproposalTimeoutTicks is the number of ticks before reproposing a Raft
 	// command. This also specifies the number of ticks between each reproposal
 	// check, so the actual timeout is 1-2 times this value.
 	RaftReproposalTimeoutTicks int
```

Comment:
If we're going to change the data type for "tick" units, let's do so everywhere.